azext_iot/digitaltwins/commands_models.py

```diff
@@ -13,11 +13,16 @@
 
 def add_models(
     cmd, name_or_hostname, models=None, from_directory=None,
-    resource_group_name=None, failure_policy=ADTModelCreateFailurePolicy.ROLLBACK.value
+    resource_group_name=None, failure_policy=ADTModelCreateFailurePolicy.ROLLBACK.value,
+    max_models_per_batch=30
```

Comment:
Setting the default here could  still be by variable reference

Reply:
Basically, make a variable to reference the 30 in common.py and reference that here and in tests :)